spec/controllers/charges_controller_spec.rb

```diff
@@ -3,4 +3,78 @@
 require 'rails_helper'
 
 RSpec.describe ChargesController, type: :controller do
+  include MockApiResponseHelper
+
+  describe "POST create" do
```

Comment:
nit single quotes instead of double. Can you run `rubocop -a`

Reply:
🏆